[BUG] AttributeError: 'NoneType' object has no attribute 'pop'

**Describe the bug**
Conditional Access backup produces `AttributeError: 'NoneType' object has no attribute 'pop'`. API permissions have been updated to include all required permissions.

**To Reproduce**
Backing up a policy with the name `Office 365 - E5; Unmanaged platforms; Browser; Use Conditional Access App Control`. My next test will be to re-name the policy to remove the `;` character.

**Expected behavior**
Backup should complete.

```log
Backing up Conditional Access policy: Office 365 - E5; Unmanaged platforms; Browser; Use Conditional Access App Control
Traceback (most recent call last):
  File "/home/runner/.local/bin/IntuneCD-startbackup", line 8, in <module>
    sys.exit(start())
  File "/home/runner/.local/lib/python3.8/site-packages/IntuneCD/run_backup.py", line 246, in start
    run_backup(args.path, args.output, exclude, token)
  File "/home/runner/.local/lib/python3.8/site-packages/IntuneCD/run_backup.py", line 210, in run_backup
    config_count += savebackup(path, output, token)
  File "/home/runner/.local/lib/python3.8/site-packages/IntuneCD/backup_conditionalAccess.py", line 36, in savebackup
    policy['grantControls'].pop('authenticationStrength@odata.context', None)
AttributeError: 'NoneType' object has no attribute 'pop'
Error: Process completed with exit code 1.
```

**Run type (please complete the following information):**
 - Mode: 0
 - Client: Pipeline
 - Version 1.2

--- a/README.md
+++ b/README.md
@@ -20,6 +20,9 @@ The package can also be run standalone outside a pipeline, or in one to only bac
 # Exciting news 📣
 The front end for IntuneCD has now been released. Check it out [here](https://github.com/almenscorner/intunecd-monitor)
 
+## What's new in 1.2.1
+- Bug fix for back up of Conditional Access throwing `AttributeError: 'NoneType' object has no attribute 'pop'` when no `grantControls` were configured.
+
 ## What's new in 1.2.0
 - Improvements to documentation
   - If a setting within a table cell has multiple entries, they are now separated by a new line instead of just a comma

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,6 @@
 [metadata]
 name = IntuneCD
-version = 1.2.0
+version = 1.2.1
 author = Tobias Almén
 author_email = almenscorner@outlook.com
 description = Tool to backup and update configurations in Intune

--- a/src/IntuneCD/backup_conditionalAccess.py
+++ b/src/IntuneCD/backup_conditionalAccess.py
@@ -33,7 +33,8 @@ def savebackup(path, output, token):
             print("Backing up Conditional Access policy: " + policy['displayName'])
 
             policy = makeapirequest(f"{ENDPOINT}/{policy['id']}", token)
-            policy['grantControls'].pop('authenticationStrength@odata.context', None)
+            if policy['grantControls']:
+                policy['grantControls'].pop('authenticationStrength@odata.context', None)
             policy = remove_keys(policy)
 
             # Get filename without illegal characters

